docs/validators/concepts/rewards.md

```
@@ -20,50 +20,50 @@ _<center>Logic similar to BTC supply, where block reward reduces so the block re
 
 <br/>
 
-**To determine $f_n$ , TN compares REP minted between 2 previous periods where, a delta ($d$) is defined for period $n$:**
+**To determine f_n, TN compares REP minted between 2 previous periods where, a delta (d) is defined for period n:**
 
 <br/>
 
-$$
-d_n = \frac{\frac{REP_{from \space period \space n-1}}{REP_{from \space period \space n-2}}}{1+g}
-$$
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)